content/cpp/concepts/math-functions/terms/nearbyint/nearbyint.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'nearbyint()'
+Description: 'Returns the argument rounded to the closest integer as a floating-point value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`nearbyint()`** function returns the argument rounded to the closest integer as a floating-point value, according to the current rounding method.
+
+## Syntax
+
+```pseudo
+nearbyint(value)
+```
+The function accepts a `value` parameter that represents a positive or negative floating-point value. The current rounding method, as described by `fegetround()`, determines which direction the `value` is rounded. By default, the rounding direction is set as `FE_TONEAREST`.
+
+### Example
+
+The following example returns the nearby integer of `1.5`:
+
+```codebyte/cpp
+#include <iostream>
+#include <cmath>
+
+int main(){
+    double value = 1.50;
+    int result;
+    result = std::nearbyint(value);
+
+    std::cout << "Nearest value is " << result << "\n";
+    // Output: Nearest value is 2
+}
+```
+
```

Comment:
```suggestion

This will produce the following output:

```

And then let's add a newline, followed by a \```shell code block that looks like this:

```shell
Nearest value is 2
```